RealmSwift/App.swift

```diff
@@ -74,14 +74,46 @@ public typealias PushClient = RLMPushClient
 /// An object which is used within UserAPIKeyProviderClient
 public typealias UserAPIKey = RLMUserAPIKey
 
-/// A `Credentials` represents data that uniquely identifies a Realm Object Server user.
-public typealias Credentials = RLMCredentials
+/**
+`Credentials`is an enum representing supported authentication types for MongoDB Realm.
+Example Usage:
+```
+let credentials = Credentials.JWT(token: myToken)
+```
+*/
+public enum Credentials {
+    /// Credentials from a Facebook access token.
+    case facebook(accessToken: String)
+    /// Credentials from a Google serverAuthCode.
+    case google(serverAuthCode: String)
+    /// Credentials from an Apple id token.
+    case apple(idToken: String)
+    /// Credentials from an email and password.
+    case emailPassword(email: String, password: String)
+    /// Credentials from a JSON Web Token
+    case JWT(token: String)
```

Comment:
This should be lowercase, thoughts @tgoyne ?

Reply:
as in `case jwt(token: String)`?

Reply:
Yes, since it's the first word in the name it's all lowercase.